web/src/components/apps/Dashboard.tsx

```diff
@@ -48,9 +48,14 @@ type Props = {
   isBundleUploading: boolean;
   isHelmManaged: boolean;
   isVeleroInstalled: boolean;
-  makeCurrentVersion: (version: Version) => void;
+  makeCurrentVersion: (
+    slug: string,
+    versionToDeploy: Version,
+    isSkipPreflights: boolean,
+    continueWithFailedPreflights: boolean
+  ) => void;
   ping: (clusterId?: string) => void;
-  redeployVersion: (version: Version) => void;
+  redeployVersion: (slug: string, version: Version | null) => void;
```

Comment:
these are just passed through to the version card. we'll need to refactor out this prop drilling eventually